frontend/index.html

```diff
@@ -201,7 +206,7 @@ <h2><a class="hash-link" href="#opt-in">Preloading Should Be Opt-In</a></h2>
       If you maintain a project that provides HTTPS configuration advice or provides an option to enable HSTS, <strong>do not include the <code>preload</code> directive by default</strong>. We get regular emails from site operators who tried out HSTS this way, only to find themselves on the preload list by the time they find they need to remove HSTS to access certain subdomains. <a href="#removal">Removal</a> tends to be slow and painful for those sites.
     </p>
     <p>
-      It's great to support HSTS preloading as a best practice, and for projects to provide a simple option to enable it. However, site operators who enable HSTS should know about the long-term consequences of preloading before they turn it on for a given domain. They should also be informed that they need to meet additional requirements and submit their site to <a href="https://hstspreload.org/">hstspreload.org</a> to ensure that it is successfully preloaded (i.e. to get the full protection of the intended configuration).
+      Site operators who enable HSTS should know about the long-term consequences of preloading before they turn it on for a given domain. They should also be informed that they need to meet additional requirements and submit their site to <a href="https://hstspreload.org/">hstspreload.org</a> to ensure that it is successfully preloaded (i.e. to get the full protection of the intended configuration).
```

Comment:
The removal of the first sentence of this paragraph looks good. In the edit, what's now the first sentence looks like it's addressed at a different audience (site operators) than this section is addressed at (projects that provide HTTPS configuration). I think the gist of this paragraph is that projects configuring HTTPS need informed consent from the site operator before enabling HSTS or HSTS preload. Is there an easy way to edit this to better align with that goal?

Reply:
Re-wrote this a bit to make the subject be about such projects (rather than directed at site owners). Let me know what you think.